src/routes/private.ts

```diff
@@ -71,6 +82,72 @@ export default async function (fastify: FastifyInstance): Promise<void> {
       return res.status(200).send(operationResult);
     },
   );
+  fastify.post<{ Body: Opportunity }>('/newOpportunity', async (req, res) => {
+    if (!req.service) {
+      return res.status(404).send();
+    }
+
+    const opportunity = opportunityCreateSchema.safeParse(req.body);
+    if (opportunity.error) {
+      req.log.error(
```

Comment:
```suggestion
      logger.error(
```

use our exported logger?

Reply:
It just generalizes the error right?
think the req has tracing?

CC @omBratteng probably knows best nuance.

Reply:
its the same config, but we use logger export all around to keep it consistent